src/main/java/uk/gov/hmcts/reform/et/syaapi/service/pdf/PdfService.java

```diff
@@ -328,14 +330,18 @@ private byte[] convertClaimantResponseToPdf(RespondToApplicationRequest request)
         );
     }
 
-    private byte[] convertClaimantTseToPdf(ClaimantTse claimantTse) throws DocumentGenerationException {
+    private byte[] convertClaimantTseToPdf(ClaimantTse claimantTse,
+                                           String contactApplicationDate,
+                                           String contactApplicant) throws DocumentGenerationException {
```

Comment:
You can move this method to another class or create a new utility class as this class has lot's of methods which should be moved.